junifer/markers/tests/test_reho_parcels.py

```diff
@@ -0,0 +1,66 @@
+"""Provide tests for ReHo on parcels."""
+
+# Authors: Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from pathlib import Path
+
+from nilearn import image as nimg
+
+from junifer.markers.reho_parcels import ReHoParcels
+from junifer.storage.sqlite import SQLiteFeatureStorage
+from junifer.testing.datagrabbers import SPMAuditoryTestingDatagrabber
+
+
+PARCELLATION = "Schaefer100x7"
+
+
+def test_reho_parcels_computation() -> None:
+    """Test ReHoParcels fit-transform."""
+    with SPMAuditoryTestingDatagrabber() as dg:
+        # Use first subject
+        subject_data = dg["sub001"]
+        # Load image to memory
+        fmri_img = nimg.load_img(subject_data["BOLD"]["path"])
+        # Initialize marker
+        reho_parcels_marker = ReHoParcels(parcellation=PARCELLATION)
+        # Fit transform marker on data
+        reho_parcels_output = reho_parcels_marker.fit_transform(
+            {"BOLD": {"data": fmri_img}}
+        )
+        # Get BOLD output
+        reho_parcels_output_bold = reho_parcels_output["BOLD"]
+        # Assert BOLD output
+        assert "data" in reho_parcels_output_bold
+        assert "columns" in reho_parcels_output_bold
+        assert "rows_col_name" in reho_parcels_output_bold
+        # TODO: add comparison with afni 3dReHo generated data
```

Comment:
How difficult is to do this before merging the PR?

Reply:
I'm working on it but there's some issue with type conversions in afni which is blocking it. And, not sure if other errors will pop up after that.